### ISSUE TITLE
Merge improvements

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#3426]
One improvement I would like concerns the second data frame being merged into the "main one".  Please could we have an option to be able to choose the columns from that data frame that are merged.  

Currently (I think) it merges all the columns.  I don't always want this.  It often makes the resulting data frame unnecessarily large.

Once this is done - or anyway - I don't find this to be an easy dialogue to use.  I wonder what others feel.  It is a very important dialogue for us, because we make a point of being able to cope well with multiple data frames.

**Comment by dannyparsons** [comment]
Working on this now.
